src/components/App.jsx

```diff
@@ -12,6 +12,7 @@ import Profile from '../containers/Profile'
 import Devices from '../containers/Devices'
 import Sessions from '../containers/Sessions'
 import Storage from '../containers/Storage'
+import Exports from '../containers/Exports'
```

Comment:
It think this change should be in a separate commit, what do you think ? Because it's not related to upgrading cozy-client.

Reply:
I think the line is wrong here. Do you mean about initialising the client? Because the upgrade itself I already did it in a separate commit.

Reply:
Actually I am talking about the fact that we are adding a new `Export` component in a commit named `upgrade cozy-client-js`.